lib/tether/tether.go

```diff
@@ -42,11 +44,16 @@ import (
 )
 
 const (
-	// The maximum number of records to keep for restarting processes
+	// MaxDeathRecords The maximum number of records to keep for restarting processes
 	MaxDeathRecords = 5
 
 	// the length of a truncated ID for use as hostname
 	shortLen = 12
+
+	// temp directory to copy existing data to mounts
+	bindDir = "/.bind"
+	// markfile to avoid re-copying existing data to mounts
+	mountsCopied = "/.mountscopied"
```

Comment:
I'm not sure this is worth implementing since we're using the same path for shared volumes.  In which case container1 and container2 and both mount the same volume (`someVol`), and both will race on copying their contents to it.

Rather, lets remove this check and always copy the dirents in the underlying mount point (the bind mount) as long as they are newer or do not exist.